Sources/App/Models/Approve/Approve.swift

```diff
@@ -0,0 +1,35 @@
+import Vapor
+import FluentProvider
+
+// sourcery: AutoModelGeneratable
+// sourcery: toJSON, Preparation, fromJSON, Updateable
+final class Approve: Model {
```

Comment:
Maybe rename it to `Approval` since `Approve` is a verb. But not necessary in fact :)

Reply:
agree, fixed.